utils/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is the `AndroidManifest.xml` file necessary? Will things still work if it's deleted?